app/controllers/home_controller.rb

```diff
@@ -1,2 +1,11 @@
 class HomeController < ApplicationController
+  def index
+    @q = Question.ransack(ransack_params)
+    @q.sorts = "title desc" if @q.sorts.empty?
+    @feed_questions = @q.result(distinct: true)
```

Comment:
why distinct?

Reply:
because questions can have multiple topics then it will come more than once